play-scala-tls-example/scripts/script-helper

```diff
@@ -1,7 +0,0 @@
-#!/usr/bin/env bash
-
-set -e
-set -o pipefail
-
-# Export the keystore password for use in ws.conf
-export KEY_PASSWORD=`cat scripts/password`
```

Comment:
This one I saw was being used.  Do you know more about it?

Reply:
Yeah, we should add it back.